.github/CODEOWNERS

```diff
@@ -32,3 +32,15 @@ tests/internal/remoteconfig         @DataDog/apm-core-python @DataDog/asm-python
 # Profiling
 ddtrace/profiling                   @DataDog/apm-core-python @DataDog/debugger-python
 tests/profiling                     @DataDog/apm-core-python @DataDog/debugger-python
+
+# Guild
+setup.py                            @DataDog/python-guild
+setup.cfg                           @DataDog/python-guild
+pyproject.toml                      @DataDog/python-guild
+.readthedocs.yml                    @DataDog/python-guild
+README.md                           @DataDog/python-guild
+mypy.ini                            @DataDog/python-guild
+.github/PULL_REQUEST_TEMPLATE.md    @DataDog/python-guild
+.github/ISSUE_TEMPLATE.md           @DataDog/python-guild
+ddtrace/internal/remoteconfig       @DataDog/python-guild
+ddtrace/internal/compat.py          @DataDog/python-guild
```

Comment:
for this and remoteconfig, should the guild also own the tests?

Reply:
Oh yes, good catch with the tests, that should be the case.

Reply:
addressed